lib/common/models/redis_store.rb

```diff
@@ -53,6 +53,10 @@ def self.find(redis_key = nil)
       end
     end
 
+    def self.find_or_create(redis_key)
+      find(redis_key) || new(Hash[@redis_namespace_key, redis_key])
```

Comment:
I think this should be find_or_build, no find_or_create

I have a similar method on:  `lib/common/client/session.rb` which inherits from RedisStore. Do you think we could just move method to RedisStore and keep it consistent?

I'm pretty sure I added specs for the method too, but can't recall -- it was a little while ago. 

def self.find_or_build(attributes)
        find(attributes[redis_namespace_key]) || new(attributes)
      end